docs/src/showcase.md

```diff
@@ -40,6 +40,8 @@ title: "Community Showcase"
 * [Testim Playground](https://www.testim.io/playground/): Record Playwright UI tests as code
 * [Try Playwright](https://try.playwright.tech/): Interactive playground for Playwright to run examples directly from your browser
 * [Applitools](https://applitools.com): Add AI-powered visual assertions and run your Playwright tests on all browser, device, and viewport combinations in parallel, without requiring any setup.
+* [Sauce Labs](https://saucelabs.com/platform/automation-tools/playwright): Run Playwright tests in parallel in the cloud or your own docker environment.
```

Comment:
On the website it say:

> Sign up now to be notified when it's ready or check back for updates.

Is this already available? since I would only add it once its fully available for use.

Reply:
It's absolutely available for use, thank you for catching this one! 🤦 🙇